extension/resources/walkthrough/plots.md

```diff
@@ -1,61 +1,82 @@
 # Plots
 
-Select one or more experiments to visualize them in the
-[`Plots Dashboard`](command:dvc.showPlots). This is the extension's equivalent
-of the `dvc plots show` and `dvc plots diff` commands.
+Select one or more experiments to visualize in the
+[**Plots Dashboard**](command:dvc.showPlots).
 
-💡 If you don't have any DVC
-[plots](https://dvc.org/doc/command-reference/plots) in the project, start
-writing data points into JSON, YAML, CSV or TSV or saving plots as images
-(`.png`, etc) (check the [DVCLive](https://dvc.org/doc/dvclive) helper library
-if you use Python):
+> This is the similar to the [`dvc plots show`] and [`dvc plots diff`] commands.
+
+[`dvc plots show`]: https://dvc.org/doc/command-reference/plots/show
+[`dvc plots diff`]: https://dvc.org/doc/command-reference/plots/diff
+
+💡 To add [DVC plots] to the project, start writing data series into JSON, YAML,
+CSV, or TSV files; or save your own plot images (`.png`, etc.).
 
 <p align="center">
   <img src="images/plots-dump-with-open-file.png"
        alt="Code to Dump a JSON Plot File" />
-  <img src="images/plots-dump-with-dvclive.png"
-       alt="Code to Dump a JSON Plot File with DVCLive" />
   <img src="images/plots-dump-image.png"
        alt="Code to Dump an Image Plot File" />
 </p>
 
-Use `DVC: Show Plots` from the
-[Command Palette](command:workbench.action.quickOpen?%22>DVC:%20Show%20Plots%22)
-to open up the plots dashboard. The extension will display the following
-sections for selected experiments, that correspond to the different
-[types of plots](https://dvc.org/doc/command-reference/plots#supported-file-formats)
-supported by DVC:
+If you're using Python, the [DVCLive] helper library can save plots data for
+you!
+
+<p align="center">
+  <img src="images/plots-dump-with-dvclive.png"
+       alt="Code to Dump a JSON Plot File with DVCLive" />
+</p>
+
+[dvc plots]: https://dvc.org/doc/start/experiments/visualization
+[dvclive]: https://dvc.org/doc/dvclive
+
+Use
+[`DVC: Show Plots`](command:workbench.action.quickOpen?%22>DVC:%20Show%20Plots%22)
+from the command palette to open the **Plots Dashboard**. It displays the
+following types of plots (for the selected experiments).
+
+## Data Series
 
 <p align="center">
   <img src="images/plots-data-series.png"
        alt="Plots: Data Series" />
 </p>
 
-`Data Series`. JSON, YAML, CSV or TSV files visualized using the predefined
-(e.g. confusion matrix, linear) or custom (Vega-lite) templates.
+JSON, YAML, CSV, or TSV files graphed using [plot templates], which may be
```

Comment:
why graphed is better? I would avoid using it - it's one more quite specific term - plot, graph, chart - all of them have their own specific meaning, we are abusing plots to some extent but at least we are not defining plots through graph yet. These explanations should be as human readable as possible.

(to be clear the current one - I don't like it either, just didn't have enough time to iterate, and not sure it can be that much simplified further - that's why I had to go with some examples even)

Reply:
"graphed" seems better (more specific) than the current "visualized" to me. "plotted using plot templates" seems repetitive.  No strong opinion, feel free to commit:

```suggestion
JSON, YAML, CSV, or TSV files visualized using [plot templates], which may be
```